### DISCUSSION TITLE
fetch page views from Web Analytics in sveltekit

### DISCUSSION THREAD

**Discussion by islamzaoui** [vercel/community#4621] [Help] [answered]
### Summary

Hello guys
I want to fetch views count from the Web Analytics of my svelte-kit site
is there any way to do that?

### Steps to Reproduce

I want something like this or similar to fetch data:
```typescript
    const pageViews = await fetch(`/webanalyticsapi?page=${currentPagePath}`,{....})
```

**Comment by amyegan** [comment, marked as answer]
Web Analytics doesn't have a public API yet, but new features will be posted to the [changelog](https://vercel.com/changelog) when they are released. You can follow that for the latest updates.
